### PR TITLE
Pattern normalization

### PR DESCRIPTION
**Pull request by tbw777**
More shortest and simple patterns.

### CODE REVIEW

File: maven-model-builder/src/main/java/org/apache/maven/model/validation/DefaultModelValidator.java [hunk 1]
```diff
@@ -68,7 +68,7 @@
 @Singleton
 public class DefaultModelValidator implements ModelValidator {
 
-    private static final Pattern CI_FRIENDLY_EXPRESSION = Pattern.compile("\\$\\{(.+?)\\}");
+    private static final Pattern CI_FRIENDLY_EXPRESSION = Pattern.compile("\\$\\{(.+?)}");
```
Comment (michael-o) [inline maven-model-builder/src/main/java/org/apache/maven/model/validation/DefaultModelValidator.java:71]: Why unbalanced?

Comment (tbw777) [reply maven-model-builder/src/main/java/org/apache/maven/model/validation/DefaultModelValidator.java:71]: I think first entry is escaping

Comment (michael-o) [reply maven-model-builder/src/main/java/org/apache/maven/model/validation/DefaultModelValidator.java:71]: I guess so as well.

**Comment by michael-o** [comment]
Can you back port this along with the other issue to 3.8.x? I will merge this one into master and 3.9.x.

**Comment by michael-o** [comment]
Please provide also a PR for master as well.

**Comment by tbw777** [comment]
@michael-o 
#990

**Comment by michael-o** [comment]
> @michael-o master: #990
> 
> backport 3.8.x #992 [[MNG-7686](https://issues.apache.org/jira/browse/MNG-7686)]

Thanks, will try to handle ASAP.

**Review by michael-o — APPROVED** [review]

**Comment by michael-o** [comment]
Merged.

**Pull request closed without merging**